### PR TITLE
Fix deleting users route

### PR DESCRIPTION
**Pull request by runnersaw**

### CODE REVIEW

**Review by rvanderheyde — APPROVED** [review]
Looks good. Could also check if user is not empty to make sure the remove actually did something

File: routes/team.js [hunk 1]
```diff
@@ -28,4 +29,15 @@ team.post("/data", function(req, res) {
 	});
 });
 
-module.exports = team;
\ No newline at end of file
+team.post("/delete", authHelper.isAdmin, function(req, res) {
```
Comment (cwallac) [inline routes/team.js:32]: Rather than posting to a delete endpoint you could delete to a user endpoint, its more inline with REST practice

Comment (rvanderheyde) [reply routes/team.js:32]: Chris is right that is more RESTful.

Comment (rvanderheyde) [reply routes/team.js:32]: But if it is an internal api, it doesn't really matter

Comment (cwallac) [reply routes/team.js:32]: True, but its usually easier to maintain one users endpoint that can be POSTed to and DELETEd to than have separate endpoints doing the services.

Comment (runnersaw) [reply routes/team.js:32]: @rvanderheyde Would you prefer DELETEing to /users or add some additional path? For example should it be team.delete("/", ...) or team.delete("/user", ...)?

Comment (runnersaw) [reply routes/team.js:32]: I added an item to our Trello board to update our API to be more in line with REST practices.

**Pull request merged**